File: leetcode_buggy_version/python/TwoSum.py

from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indexing = dict()
        array = [[]]
        
            
        for i in range(len(nums)):
            if target-nums[i] in indexing:
                array.append(i)
                array.append(indexing[target-nums[i]])
                return array 
            indexing[nums[i]] = i


#https://leetcode.com/problems/two-sum/solutions/5068957/two-approach-python3/

File: leetcode_buggy_version/java_bugg/TwoSum.java
package leetcode_buggy_version.java_bugg;

public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if ((nums[i] + nums[j]) == target) {
                    return nums;
                }
            }
        }
        return null;
    }
}

// https://leetcode.com/problems/two-sum/solutions/5050238/code-in-java/
// Error: return nums;
